getting-started-guide/event-data.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: "Events in Axiom"
+description: "This page explains the fundamentals of event data in Axiom."
+sidebarTitle: Events
+---
+
+In Axiom, event data is the atomic unit of activity, encompassing logs, traces, and even product interactions. Each event is simply a structured record—composed of key-value pairs—that captures meaningful interactions or changes in state within a system. While these can appear in various forms, from log lines to complex traces, they usually contain the following:
```

Comment:
Let’s call it "timestamped event data" in the first mention, and event data thereafter. I want to ground event data very clearly in time from the outset. Note: This is _not_ described as time _series_ event data. The language choice is intentional, to distance Axiom rightly from time series DBs.